test/unit/src/types/InjectReq.test.ts

```diff
@@ -0,0 +1,65 @@
+import { initAjvSchemas, verifyPayload } from '../../../../src/types/ajv/Helpers'
+
+describe('Inject req test', () => {
+    beforeAll(() => {
+        initAjvSchemas()
+    })
+
+    describe('Data validation Cases', () => {
+        const invalidObjects = [
+            {
+                // sender: '0x0',
+                user: {},
+            },
+
+        ]
+        const otherInvalidObject = [
+            {
+                timestamp: null,
+                accountId: 'c',
+            },
+            {
+                timestamp: '0x0',
+            },
+        ]
+        const validObjects = [
+            {
+                timestamp: 1719305344,
+            },
+            {
+                sender: '0x0',
+                sign: {
+                    owner: '0x1',
+                    sig: '0x1',
+                },
+                timestamp: 2,
+                accountId: '0x1',
+            },
+        ]
+        test.each(invalidObjects)('should throw AJV error', (data) => {
+            const res = verifyPayload('InjectReq', {
+                ...data,
+            })
+            console.log('res', res!)
```

Comment:
remove extra console logs, unnecessary comments from all the unit tests. Some tests fail, fix them too